### ISSUE TITLE
Variable `output directory` flags

### ISSUE THREAD

**Issue by susheelbhanu** [sourmash-bio/sourmash#1812]
Hey all.. Just wanted to bring this to your attention, but I'm running the latest (I believe) version of `sourmash`.
```== This is sourmash version 4.2.3. ==```

There is a minor discrepancy in how the `output directory` flags are called for `sourmash compute` and `sourmash plot`. See below:

1.  *compute*
```
sourmash compute --help | grep 'outdir'

== This is sourmash version 4.2.3. ==
== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==

  --outdir OUTDIR       output computed signatures to this directory
```

2. *plot*
```
sourmash plot --help | grep 'output-dir'

== This is sourmash version 4.2.3. ==
== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==

             [-f] [--output-dir DIR] [--csv F]
  --output-dir DIR      directory for output plots
```

I realise that one is for signatures and the other for plots, but it help to harmonise the two flags irrespective of the outputs. 

Thank you!

**Comment by ctb** [comment]
thank you! will fix!

**Comment by susheelbhanu** [comment]
Thank you!

**Comment by ctb** [comment]
argh, it's worse than you know! 😂
```
src/sourmash/cli/compute.py:        '--outdir', help='output computed signatures to this directory'
src/sourmash/cli/sig/split.py:        '--outdir', help='output signatures to this directory'
src/sourmash/cli/sketch/dna.py:        '--outdir', help='output computed signatures to this directory'
src/sourmash/cli/sketch/protein.py:        '--outdir', help='output computed signatures to this directory'
src/sourmash/cli/sketch/translate.py:        '--outdir', help='output computed signatures to this directory'

src/sourmash/cli/plot.py:        '--output-dir', metavar='DIR', help='directory for output plots'
src/sourmash/cli/tax/annotate.py:        '-o', '--output-dir', default= "",
src/sourmash/cli/tax/genome.py:        '--output-dir', default= "",
src/sourmash/cli/tax/metagenome.py:        '--output-dir', default= "",
```

**Comment by susheelbhanu** [comment]
And here I thought, it will be simple task 🙈. Good luck but thank you for synchronizing it! 🙏🏽

**Comment by ctb** [comment]
eh, I'm just adding aliases :). easy enough!

over in https://github.com/sourmash-bio/sourmash/pull/1817.

**Comment by ctb** [comment]
Merged in https://github.com/sourmash-bio/sourmash/pull/1817. Thanks again!

**Comment by ctb** [comment]
Released in sourmash v4.3.0.
